base/pkg/entry.jl

```diff
@@ -340,11 +344,14 @@ function update(branch::AbstractString)
             end
         end
         LibGit2.fetch(repo)
-        LibGit2.merge!(repo, fastforward=true) || LibGit2.rebase!(repo, "origin/$branch")
+        rebased = LibGit2.merge!(repo, fastforward=true)
```

Comment:
that doesn't sound right to me. a fastforward is a trivial merge where the working copy was unchanged from a past state of the merge branch, doesn't seem like it has anything to do with rebase, there aren't intermediate commits other than the new merged ones, and there's no squashing?